styles/styles.css

```diff
@@ -6,3 +6,148 @@
  * 
  *
  */
+
+/* main > div {
+  width: 100%!important;
+  margin: 0 auto;
+  max-width: 1260px;
+  padding-left: 70px!important;
+  padding-right: 70px!important;
+} */
+
+body {
+  line-height: 21px;
+}
+
+body table {
+  font-size: 16px;
+  border-spacing: 0;
+}
+
+table, th, td {
+  padding: 9px 10px;
+  border: 1px solid #bdbdbd!important;
+  border-collapse: collapse!important;
+}
+
+table th {
+  background-color: #ededed;
+}
+
+.columns {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.accordion dd div {
+  padding-top: 10px;
+}
+
+.content {
+  display: block;
+  max-width: var(--grid-container-width);
+  margin: 0 auto;
+}
+
+/*
+* title
+*/
+
+.page-title {
+  padding: 10px!important;
+  font-size: 36px;
+  font-weight: 500;
+  color: white!important;
+}
+
+/*
+  internal banner
+*/
+
+main .internal-banner .banner:before {
```

Comment:
My understanding is that only global styles should be present here.
Could you please check whether styles related to `internal-banner` should go here.

Reply:
Yes, they should be here as internal banner has been implemented as a global component in https://github.com/adobecom/helpx-internal/blob/global-blocks-and-styles/scripts/scripts.js#L160-L187